test/unit-tests/vast_test.js

```diff
@@ -1721,4 +1761,281 @@ describe('ad_manager_vast', function() {
     vastAdManager.loadMetadata({"tagUrl": "http://blahblah"}, {}, content);
     expect(vastAdManager.vastUrl).to.be("http://blahblah");
   });
+
+  it('VPAID 2.0: Should parse VPAID linear creative', function() {
+    vpaidInitialize();
+    var ad = amc.timeline[0];
+    expect(ad).to.be.an('object');
+    expect(ad.duration).to.eql(16);
```

Comment:
since we have number values coming through. Are you checking if string values work as well?  So <myTag duration=15>  and <myTag duration="15">